presto-native-execution/CMakeLists.txt

```diff
@@ -188,6 +188,9 @@ include_directories(velox/velox/external/xxhash)
 include_directories(${VELOX_ROOT})
 include_directories(${CMAKE_BINARY_DIR})
 
+# also add nlohmann/json external
+include_directories(presto_cpp/external/json)
```

Comment:
Let's add this only to the JWT dependency.

Reply:
Ok, removed it again. We will use the directive when adding the JWT.